src/bundle/Resources/public/scss/_mixins.scss

```diff
@@ -233,3 +229,8 @@
     border-bottom-color: $ibexa-color-primary;
     border-left-color: $ibexa-white;
 }
+
+@mixin ibexa-font-sans-serif() {
```

Comment:
Please do not add new mixins in this file.
Please create a file in `/mixins` and import it in this file.